.gitignore

```diff
@@ -26,4 +26,3 @@ __*
 *.coverprofile*
 .DS_Store
 
```

Comment:
s2ioperator is generated and changed frequently. Such file should not keep in repo

Reply:
s2ioperator.yaml in the git repository, adding to the gitignore will be easy to misunderstand.
So I deleted the ignore

Reply:
you can delete the yaml in this pr